doc-examples/hello-world-java/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id 'java'
+    id 'com.google.protobuf' version '0.8.17'
+    id 'io.micronaut.application' version '2.0.6'
+}
+
+ext {
+    micronautVersion = '3.1.1'
+    protobufJavaVersion = '3.17.2'
+    grpcVersion = '1.39.0'
+}
+
+repositories {
+    mavenCentral()
+}
+
+mainClassName = "helloworld.Application"
+micronaut {
+    version micronautVersion
+    testRuntime "junit5"
+    enableNativeImage false
+}
+
+dependencies {
+
+    annotationProcessor "io.micronaut:micronaut-inject-java"
+    annotationProcessor "io.micronaut:micronaut-validation"
+
+    compileOnly "io.grpc:grpc-stub:$grpcVersion"
+    compileOnly "io.micronaut:micronaut-inject-java"
+    implementation "io.micronaut:micronaut-inject"
+    implementation "io.micronaut:micronaut-validation"
+
+    implementation "io.micronaut.grpc:micronaut-grpc-server-runtime"
+
+    implementation "io.micronaut.discovery:micronaut-discovery-client"
+    runtimeOnly "ch.qos.logback:logback-classic"
+
+    testAnnotationProcessor "io.micronaut:micronaut-inject-java"
+    testAnnotationProcessor "io.micronaut:micronaut-validation"
+
+    testImplementation "org.junit.jupiter:junit-jupiter-api"
+    testImplementation "io.micronaut.test:micronaut-test-junit5"
+    testImplementation "io.micronaut:micronaut-inject-java"
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine"
+}
```

Comment:
You don't need all these dependencies, specially the test and others like inject-java. The Micronaut gradle plugin already adds them. Please take a look at a fresh application from https://launch.micronaut.io/